battery_level_percent: 20.0
take_off_time_sec: 10.0
max_vel_:  1.0
min_vel_: -1.0
